app/js/services/simulationService.js

```diff
@@ -1,54 +1,37 @@
 'use strict';
 
 angular.module('ddsApp').service('SimulationService', function($http, droitsDescription) {
-    function makeObjectFilter(test) {
-        return function(source) {
-            var result = {};
 
-            _.forEach(source, function(value, key) {
-                if (test(value, key)) {
-                    result[key] = value;
-                }
-            });
-
-            return result;
-        };
-    }
-
-    var filterUnhandled = makeObjectFilter(function(value, key) {
-        return droitsDescription[key];
-    });
-
-    /* jshint unused: false */
-    var filterIneligible = makeObjectFilter(function(value, key) {
-        return value;
-    });
-
-    function describe(droits) {
-        return _.mapValues(droits, function(montant, key) {
-            var result = _.clone(droitsDescription[key]);
-            result.montant = montant;
-            return result;
+    // Si la valeur renvoyée par l'API est nulle, cela signifie par convention que l'aide a été injectée et non recaculée par le simulateur
+    function sortDroits(droitsCalcules) {
+        var droitsEligibles = {};
+        var droitsInjectees = {};
+        var droitsNonEligibles = {};
```

Comment:
Avantage, [baisse de risque](https://jslinterrors.com/combine-this-with-the-previous-var-statement) ou [inconvénient](http://stackoverflow.com/questions/694102/declaring-multiple-variables-in-javascript), c'est surtout un [choix](http://programmers.stackexchange.com/questions/229694/multiple-var-statements-in-javascript)  ;)  Personnellement, je préfère car ça enlève du bruit visuel.